iosHyperskillApp/iosHyperskillApp/Sources/Modules/StepQuiz/Views/StepQuizView.swift

```diff
@@ -125,6 +125,13 @@ struct StepQuizView: View {
                 if let formattedStats {
                     StepQuizStatsView(text: formattedStats)
                 }
+
+                Divider()
```

Comment:
Move the divider to if condition:
```swift
if viewModel.stepRoute is StepRouteLearn {
  Divider()
  ProblemsLimitAssembly().makeModule()
}
```

Reply:
I think it's better to leave it outside of if condition to make step quiz screens consistent and same. Calling `ProblemsLimitAssembly` doesn't guarantee that widget will be shown. In this case we will have divider on learn steps and don't have it on repetition and daily.

Reply:
We only need to show this divider if the widget will be shown, also paid users should not see this divider and widget